From 1550e70a4e028da4406c44c8a8dd6018459644d0 Mon Sep 17 00:00:00 2001
From: Cole Robinson <crobinso@redhat.com>
Date: Thu, 8 Feb 2018 14:23:11 -0500
Subject: [PATCH 27/45] uitests: Silence noisy harmless warnings

Could mean a code error but more likely is just slightly racy
UI bits disappearing that we weren't looking for anyways, so hide
it behind a debug statement
---
 tests/uitests/utils.py | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/tests/uitests/utils.py b/tests/uitests/utils.py
index 63e3ad08..e481e443 100644
--- a/tests/uitests/utils.py
+++ b/tests/uitests/utils.py
@@ -174,7 +174,9 @@ class _FuzzyPredicate(dogtail.predicate.Predicate):
                 return
             return True
         except Exception as e:
-            print("got predicate exception: %s" % e)
+            logging.debug(
+                    "got predicate exception name=%s role=%s labeller=%s: %s",
+                    (self._name, self._roleName, self._labeller_text, e))
 
 
 def check_in_loop(func, timeout=2):
-- 
2.16.2

